R/TuneControl.R

```diff
@@ -47,8 +47,10 @@
 #' @param tune.threshold.args [\code{list}]\cr
 #'   Further arguments for threshold tuning that are passed down to \code{\link{tuneThreshold}}.
 #'   Default is none.
-#' @param log.fun [\code{function} | \code{NULL}]\cr
-#'   Function used for logging. If set to \code{NULL}, the internal default will be used.
+#' @param log.fun [\code{function} | \code{character(1)}]\cr
+#'   Function used for logging. If set to \dQuote{default}, the internal default will be used.
+#'   If set to \dQuote{memory}, a slightly extended version of the default will be used that also displays
+#'   memory usage.
```

Comment:
It wasn't a template before (when it already was a duplicate). Is there a reason it was not?